nautobot/core/templates/components/panel/footer_table.html

```diff
@@ -0,0 +1,17 @@
+{% block footer_left %}{% endblock %}
+<div class="pull-right">
+    {% block footer_right %}
+        {% if body_content_table_add_url %}
+        <a href="{{ body_content_table_add_url }}" class="btn btn-xs btn-primary">
+            <span class="mdi mdi-plus-thick" aria-hidden="true"></span> Add {{ body_content_table_verbose_name }}
+        </a>
+        {% endif %}
+
+        {% if body_content_table_list_url and more_queryset_count > 0 %}
+        <a href="{{ body_content_table_list_url }}" class="btn btn-default btn-xs">
+            <span class="mdi mdi-database-search-outline" aria-hidden="true"></span> View {{ more_queryset_count }} more object(s)
+        </a>
+        {% endif %}
+    {% endblock %}
+</div>
+<div class="clearfix"></div>
```

Comment:
Is the `clearfix` div actually needed here? I've never been real clear what it actually does, especially within a panel.